src/main/java/com/ioet/bpm/people/boundaries/PersonController.java

```diff
@@ -52,6 +42,23 @@
                 .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
+    @ApiOperation(value = "Return one person by email or all persons", response = Person.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Person(s) successfully returned")
+    })
+    @GetMapping(produces = "application/json")
+    public ResponseEntity<?> findPersonByEmail(@RequestParam(value = "email", required = false) String email) {
```

Comment:
We should move all the logic in here to an additional class. I'd say something called PeopleService with a method name. findByPeopleByEmail(String email)

Inside that class you can do the verification to check if the email is null or not and then execute the proper query.

Reply:
We already have a PersonService class... wouldn't that be a bit confusing to have a PeopleService as well? Maybe I could somehow put this logic into the PersonService class :thinking:

Reply:
if that makes sense :joy:

Reply:
should look better now... tell me what you think!

Reply:
ok